client/src/app/domain/models/poll/poll-constants.ts

```diff
@@ -160,7 +174,7 @@ export const PollValues: any = {
     amount_global_abstain: _(`General abstain`)
 };
 
-export type PollPropertyVerboseKey = keyof typeof PollPropertyVerbose;
+// export type PollPropertyVerboseKey = keyof typeof PollPropertyVerbose;
```

Comment:
```suggestion
export type PollPropertyVerboseKey = keyof typeof PollPropertyVerbose;
```

Reply:
Why not delete it?

Reply:
Anyway what was wrong with the interface solution?